libtransmission/utils.h

```diff
@@ -85,8 +87,12 @@ constexpr auto tr_saveFile(std::string_view filename, ContiguousRange const& x,
 /** @brief return the current date in milliseconds */
 [[nodiscard]] uint64_t tr_time_msec();
 
-/** @brief sleep the specified number of milliseconds */
-void tr_wait_msec(long int delay_milliseconds);
+/** @brief sleep for the specified duration */
+template<class rep, class period>
+void tr_wait(std::chrono::duration<rep, period> const& delay)
```

Comment:
nit: Maybe move definition to `utils.cc`?

Reply:
A smaller, no-new-dependencies template function like this makes more sense in the header IMO